demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2519,7 +2523,7 @@ export class DemoMeetingApp
 
   async populateAudioInputList(): Promise<void> {
     const genericName = 'Microphone';
-    let additionalDevices = ['None', '440 Hz', 'Prerecorded Speech', 'Echo'];
+    let additionalDevices = ['None', '440 Hz', 'Prerecorded Speech', 'Echo', 'No Audio'];
```

Comment:
In description you mentioned this does not work in firefox, is it due to firefox not supporting the dependent WebRTC metric?

If thats the case, should we populate this based on browser now? We do this for simulcast, video priority options while joining the demo hence checking.

Reply:
The metric is supported by Firefox, but it's not working as expected when I change the device to `No Audio`  and I suspect the issue is with Firefox's implementation of getStats() and I'm planning to confirm that and report a bug to them.

But I don't fully get your point about populating "this" based on browser, do you mean adding `No Audio` to the additionalDevices based on browser?

Reply:
Yup. I thought that this is not supported in FF hence suggested to populate it based on browser but if it is supported, then, it would be good to check the issue and file a bug with FF directly.

Reply:
What is the different between `No Audio` and `None` ?